src/library/Form/styled.js

```diff
@@ -67,7 +67,7 @@ export const FormFieldsetRoot = styled('fieldset')(
       ...componentStyleReset(baseTheme),
 
       border: '1px solid transparent',
-      borderTopColor: theme.FormFieldset_borderColor,
+      // borderTopColor: theme.FormFieldset_borderColor,
```

Comment:
I think with the current formula we would keep this as is and just change this on our theme object. 
Something like `FormFieldset_borderColor: 'transparent'`

Reply:
👍

Reply:
That's correct. Prefer to explicitly unset it to either `undefined` or `transparent`.